README.md

```diff
@@ -128,8 +122,8 @@ To compile SASS within an ember-cli addon, there are a few additional steps:
 
 1. Include your styles in `addon/styles/addon.scss`.
 
-2. Ensure you've installed `ember-cli-sass` under `dependencies` in your
-   `package.json`.
+2. Ensure you've installed `ember-cli-sass` and either `sass` or `node-sass`
+   under `dependencies` in your `package.json`.
 
 3. Define an `included` function in your app:
    ```js
```

Comment:
It seems like we should be encouraging users to pass in a Sass implementation here, but I'm not sure how.